src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/OverloadMethodsDeclarationOrderCheck.java

```diff
@@ -29,16 +29,29 @@
 
 /**
  * <p>
- * Checks that overload methods are grouped together.
+ * Checks that overload methods are grouped together. Overloaded methods have same
+*  name but different signatures where the signature can differ by the number of
+*  input parameters or type of input parameters or both.
```

Comment:
indentation of these two lines should be 1 more

Reply:
ok, I will update

Reply:
This is updated.